tests/framework/PostProcessors/SparseSensing/testSPSLOptiTwistConstraints.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>tests/framework/Postprocessors/SparseSensing</name>
+    <author>Niharika Karnik (@niharika2999), Mohammad Abdo (@Jimmy-INL)</author>
+    <created>2024-04-01</created>
+    <classesTested>Models.Postprocessors.SparseSensing</classesTested>
+    <description>This test aims to check the ability of the Sparse Sensing Postprocessor
+     to locate and plot the selected sensors to reconstruct the temperature field of the OPTI-TWIST prototype
+     when perturbing the heater power in the presence of constraints. We assume the region near the heater to have very limited sensing capability and thus accomodate no sensors.
+    </description>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>reconstructionConstraintsOptiTwist</WorkingDir>
+    <Sequence>LoadCSV, mySPpp, print</Sequence>
```

Comment:
@niharika2999 , the reason tests are failing is that the path is inconsistent. For instance, the working directory here is  `reconstructionConstraintsOptiTwistCircle ` but in tests it is reconstructionOptiTwistConstraintsCircle. Hope this helps